android/src/main/java/com/expensify/livemarkdown/MarkdownRange.java

```diff
@@ -0,0 +1,18 @@
+package com.expensify.livemarkdown;
+
+public class MarkdownRange {
+  public String type;
+  public int start;
+  public int end;
+  public int length;
+  public int depth;
```

Comment:
```suggestion
  public final String type;
  public final int start;
  public final int end;
  public final int length;
  public final int depth;
```

Reply:
This is not resolved

Reply:
Forgot to push some changes, now it should be resolved